config.toml

```diff
@@ -2,3 +2,4 @@ EmailUsername = ""
 EmailPassword = ""
 IBMUsername = ""
 IBMPassword = ""
+Debug = false
```

Comment:
I think we want the default option to be the safe option (in case someone doesn't understand what this does). Right now, debug only affects log behavior but in the future it could turn off some security features of the app (this is how flask-base works, for example).